Makefile

```diff
@@ -12,21 +12,21 @@
 SHELL := /bin/bash
 
 export GOPATH=$(shell go env GOPATH)
-export PATH:=$(HOME)/google-cloud-sdk/bin:$(PATH)
 
 # WPTD_PATH will have a trailing slash, e.g. /home/user/wpt.fyi/
 WPTD_PATH := $(dir $(abspath $(lastword $(MAKEFILE_LIST))))
 WPTD_GO_PATH ?= $(GOPATH)/src/github.com/web-platform-tests/wpt.fyi
-WEBDRIVER_PATH ?= $(WPTD_GO_PATH)/webdriver
-BROWSERS_PATH ?= $(HOME)/browsers
-SELENIUM_PATH ?= $(BROWSERS_PATH)/selenium
-FIREFOX_PATH ?= $(BROWSERS_PATH)/firefox/firefox
-GECKODRIVER_PATH ?= $(BROWSERS_PATH)/geckodriver
+NODE_SELENIUM_PATH=$(WPTD_PATH)webapp/node_modules/selenium-standalone/.selenium/
```

Comment:
nit: Is there a reaons we use `FOO=bar` but `FOO ?= bar`? @Hexcles and @arobins do you know any make details for that should tend towards one or the other consistent style?